tiledb/sm/query/writers/global_order_writer.cc

```diff
@@ -78,6 +78,7 @@ GlobalOrderWriter::GlobalOrderWriter(
     bool disable_checks_consolidation,
     Query::CoordsInfo& coords_info,
     URI fragment_uri,
+    optional<std::string> fragment_base_name,
```

Comment:
I don't quite understand why we need this..

Reply:
The `fragment_uri` is the full fragment path for the fragment. I want to manually set just the last part of the `fragment_uri`. The `fragment_base_name` is used to do this.

I could have over-ridden the `fragment_uri` earlier, and not added another variable in. However, that would have leaked internal details on how the fragments are named to other parts of the code base.